docs/localization/LOCALIZATION.md

```diff
@@ -74,343 +80,430 @@ export function localizationConfigurationFactory() {
 ```
 
-If you leave **endPointUrl blank** the loader will try to fetch translation bundles from local src/assets/assets-otter/i18n folder.
+If you leave the property `endPointUrl` **blank** in the localization configuration, the loader will try to fetch translation bundles from your application's `src/assets/assets-otter/i18n` folder.
 
-You can specify a **language** in the configuration, it will be loaded at the bootstrap time of you application.
-A language can be specified asynchronously by using the **LocalizationService.useLanguage** function.
+You can specify a **language** in the configuration, it will be loaded at the bootstrap time of your application.
+A language can be specified asynchronously by using the `LocalizationService.useLanguage(...)` function.
 
-- In case it is not enough to provide your configuration object as explained above, you can also provide a new factory for the localization configuration. You only need to provide a token `LOCALIZATION_CONFIGURATION_TOKEN` with you custom factory and dependencies.
+- In case it is not enough to provide your configuration object as explained above, you can also provide a new factory for the localization configuration.
+  You only need to provide a token `LOCALIZATION_CONFIGURATION_TOKEN` with your custom factory and dependencies.
 
 ```typescript
-// index.ts
-
-import { LOCALIZATION_CONFIGURATION_TOKEN, LocalizationModule, translateLoaderProvider } from  "@o3r/localization";
+// app.module.ts
 
-import {BidiModule} from '@angular/cdk/bidi';
-import { TranslateModule } from  "@ngx-translate/core";
-...
-import localeAR from  "@angular/common/locales/ar";
-import localeEN from  "@angular/common/locales/en";
-import localeFR from  "@angular/common/locales/fr";
-
-
-...
+import localeAR from '@angular/common/locales/ar';
+import localeEN from '@angular/common/locales/en';
+import localeFR from '@angular/common/locales/fr';
+import { TranslateModule } from '@ngx-translate/core';
+import { LOCALIZATION_CONFIGURATION_TOKEN, LocalizationModule, translateLoaderProvider } from '@o3r/localization';
+// ...
 
 registerLocaleData(localeEN, 'en-EN');
 registerLocaleData(localeFR, 'fr');
 registerLocaleData(localeAR, 'ar');
 
 @NgModule({
   imports: [
-  LocalizationModule,
-  BidiModule,
-  TranslateModule.forRoot({
-    loader: translateLoaderProvider,
-  }),
-  ...
+    LocalizationModule,
+    TranslateModule.forRoot({
+      loader: translateLoaderProvider,
+    })
+    // ...
   ],
-  ...
+  // ...
   providers: [
     {provide: LOCALIZATION_CONFIGURATION_TOKEN, useFactory: customCreateLocalizationConfiguration, deps: [YourServiceNeededByTheFactory]}
   ]
+})
+class AppModule {}
 ```
 
-You may also need to set up your http server to accept **CORS** and update **connect-src** property of **csp.json** in your application.
+## HTTP server
 
-```typescript
+You may also need to set up your http server to accept **CORS** and update the `connect-src` property of `csp.json` in your application.
```

Comment:
Do we really need to add that in our documentation ?
It's not related to Otter no ?

Reply:
It's not related to Otter but it could be useful during the setup of the localization. 
What does everyone think?

Reply:
It's good to add it if it helps the setup
`You may also need` -> it is not clear in the doc when and why this would be needed